recipe/build.sh

```diff
@@ -17,7 +17,9 @@ if [ -z "$CC" ]; then
 fi
 
 make -j$CPU_COUNT CC="$CC" LDFLAGS="$LDFLAGS" CFLAGS="$CFLAGS"
-make test
+if [ "${CONDA_BUILD_CROSS_COMPILATION}" -ne "1" ]; then
+    make test
+fi
```

Comment:
Since `make test` fails when cross-compiling (since it is not on the right architecture), this is skipped in that case